tests/libcxx/expected_results.txt

```diff
@@ -932,13 +932,13 @@ std/containers/sequences/forwardlist/forwardlist.cons/deduct.pass.cpp FAIL
 std/containers/sequences/vector/vector.cons/deduct.pass.cpp FAIL
 
 # Not yet analyzed. Seems to force a sign conversion error?
-std/iterators/iterator.primitives/iterator.operations/advance.pass.cpp FAIL
+std/iterators/iterator.primitives/iterator.operations/advance.pass.cpp SKIPPED

```

Comment:
This is unfortunate, since `SKIPPED` means "don't tell us if upstream fixes the test." I wonder if we should add yet-another-custom-tag `WARN` to cover this case?

Reply:
Add #2625 to add such a tag to the test runner.

Reply:
Thanks for filing the followup issue (I am not sure whether it is worth adding custom machinery but now we won't forget).